pubtools/pulplib/_impl/client/client.py

```diff
@@ -179,6 +182,42 @@ def search_repository(self, criteria=None):
             response_f, lambda data: self._handle_page(Repository, search, data)
         )
 
+    def upload_one_file(
+        self,
+        file_name,
+        repo_id,
+        type_id,
+        unit_key,
+        unit_metadata=None,
+        override_config=None,
+    ):
+        """Including four steps uploading a file:
```

Comment:
Can the doc string please be formatted so that it'll render properly, and document all the arguments in the same style as done for other methods?

Overall, I agree with the idea of having a method covering these 4 steps.

Reply:
Okay.